src/pages/workspaces/workspace/useWorkspace.ts

```diff
@@ -111,8 +114,16 @@ export const useWorkspace = (namespace, name): WorkspaceDetails => {
       } else {
         updateWorkspaceInStore(workspace, false);
         console.log('Google permissions are still syncing'); // eslint-disable-line no-console
+        if (times === 1) {
```

Comment:
Don't we want `>= 1`?

Reply:
We want the event to fire exactly once per workspace load.